modules/search/pages/create-child-field.adoc

```diff
@@ -26,79 +26,16 @@ To add or remove a child field from a Search index with the Couchbase Server Web
 . Expand menu:Customize Index[Mappings]. 
 . Point to an existing mapping and click btn:[+].
 . Click *insert child field*. 
-. In the *Field* field, enter the name of a field in your documents that you want to add or remove from the index.
+. [[field]]In the *Field* field, enter the name of a field in your documents that you want to add or remove from the index.
 +
 NOTE: The field must contain either a single value or an array. If the field contains a JSON object, xref:create-child-mapping.adoc[Create a Child Mapping], instead.  
```

Comment:
```suggestion
NOTE: The field must contain either a single value or an array. If the field contains a JSON object, xref:create-child-mapping.adoc[create a child mapping] instead.  
```

Reply:
Super nitpicky. I realize you didn't originally write this - I'll focus on the new content you added for the rest of the review :)